{"name": "test_start[autotest7]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = assert_handle({'code': '6009', 'data': None, 'msg': 'Get null result of that RPC request PlatForm. Please retry later. '}, '{\\n    \"code\": \"200\",\\n    \"data\": null,\\n    \"msg\": \"success\"\\n}', 'json')", "trace": "self = <test_01.Testdata object at 0x0531BD30>\nautotest = {'Actual': '', 'Auth': '', 'Automated': 'Yes', 'CASE_NO_ID': 'test_fact_008', ...}\nwrite_book = <tools.ExcelHandle.ExcelHandle object at 0x049B8F50>\n\n    def test_start(self,autotest,write_book):\n    \n        if autotest.get('row', None):\n            response  = RequestHandle(autotest).get_response()\n            actual_result = response.json()\n            row = autotest['row']\n            result = 'Pass'\n            try:\n                assert assert_handle(actual_result,autotest['Expect'],autotest['Validate_Type'])\n            except Exception as e:\n                result = 'Fail'\n                raise AssertionError(e)\n            finally:\n                write_book.write_back_result(row,str(actual_result),result)\n        else:\n            act_data = autotest['steps']\n            for autotest in act_data:\n                response = RequestHandle(autotest).get_response()\n                row = autotest['row']\n                result = 'Pass'\n                actual_result = None\n                if response:\n                    actual_result = response.json()\n                    try:\n>                       assert assert_handle(actual_result, autotest['Expect'], autotest['Validate_Type'])\nE                       AssertionError: assert False\nE                        +  where False = assert_handle({'code': '6009', 'data': None, 'msg': 'Get null result of that RPC request PlatForm. Please retry later. '}, '{\\n    \"code\": \"200\",\\n    \"data\": null,\\n    \"msg\": \"success\"\\n}', 'json')\n\ntests\\test_01.py:40: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_01.Testdata object at 0x0531BD30>\nautotest = {'Actual': '', 'Auth': '', 'Automated': 'Yes', 'CASE_NO_ID': 'test_fact_008', ...}\nwrite_book = <tools.ExcelHandle.ExcelHandle object at 0x049B8F50>\n\n    def test_start(self,autotest,write_book):\n    \n        if autotest.get('row', None):\n            response  = RequestHandle(autotest).get_response()\n            actual_result = response.json()\n            row = autotest['row']\n            result = 'Pass'\n            try:\n                assert assert_handle(actual_result,autotest['Expect'],autotest['Validate_Type'])\n            except Exception as e:\n                result = 'Fail'\n                raise AssertionError(e)\n            finally:\n                write_book.write_back_result(row,str(actual_result),result)\n        else:\n            act_data = autotest['steps']\n            for autotest in act_data:\n                response = RequestHandle(autotest).get_response()\n                row = autotest['row']\n                result = 'Pass'\n                actual_result = None\n                if response:\n                    actual_result = response.json()\n                    try:\n                        assert assert_handle(actual_result, autotest['Expect'], autotest['Validate_Type'])\n                    except Exception as e:\n                        result = 'Fail'\n>                       raise AssertionError(e)\nE                       AssertionError: assert False\nE                        +  where False = assert_handle({'code': '6009', 'data': None, 'msg': 'Get null result of that RPC request PlatForm. Please retry later. '}, '{\\n    \"code\": \"200\",\\n    \"data\": null,\\n    \"msg\": \"success\"\\n}', 'json')\n\ntests\\test_01.py:43: AssertionError"}, "parameters": [{"name": "autotest", "value": "{'project_name': 'scello', 'file_name': 'scello.xlsx', 'sheet_name': 'Sheet1', 'steps': [{'row': 16, 'botActId': 'test066355', 'CASE_NO_ID': 'test_fact_008', 'Description': 'reload版本话术库', 'Priority': 'P1', 'Component': 'KGMS', 'Automated': 'Yes', 'Steps': 'Step1', 'Headers': '{\"Content-Type\": \"application/json\"}', 'Method': 'GET', 'Auth': '', 'Payload': '', 'URL': '/editor/reload/version/lena_de_new?resourceId=0x297', 'Validate_Type': 'json', 'Expect': '{\\n    \"code\": \"200\",\\n    \"data\": null,\\n    \"msg\": \"success\"\\n}', 'Actual': ''}], 'botActId': 'test066355'}"}], "start": 1600227109849, "stop": 1600227109903, "uuid": "9545638c-dbcb-458a-bc5c-fafbcc881101", "historyId": "4588aaa977537b0cc8801e710353d178", "testCaseId": "68de03e34e38ed87d9781455db5f577a", "fullName": "tests.test_01.Testdata#test_start", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_01"}, {"name": "subSuite", "value": "Testdata"}, {"name": "host", "value": "wx0liucy"}, {"name": "thread", "value": "10888-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_01"}]}